src/Generator/SchemaMapperGenerator.php

```diff
@@ -9,12 +9,15 @@
 use DoclerLabs\ApiClientGenerator\Naming\SchemaMapperNaming;
 use DoclerLabs\ApiClientGenerator\Output\Php\PhpFileCollection;
 use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Stmt;
```

Comment:
this is not being used, is it?

Reply:
It is used as return typehint for generateMapStatementsForFreeFormObject()